installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -330,7 +361,7 @@ int bootstrapper(HINSTANCE hInstance)
     close_progressbar_window();
 
     // Always skip dotnet install, because we should've installed it from here earlier
-    std::wstring msiProps = L"SKIPDOTNETINSTALL=1 ";
+    std::wstring msiProps = L"SKIPDOTNETINSTALL=1 " + installFolderProp;
```

Comment:
@yuyoyuppe 
does the msi still try to install .NET Core? If yes please open an issue to track removing it.